angelgolfer-msDocset0302200349/concepts/integrate_with_onenote.md

```diff
@@ -6,29 +6,34 @@ OneNote is a digital notebook that lets customers track ideas and notes for home
 
 By integrating your apps with OneNote, you can create empowering experiences across multiple platforms that reach millions of users worldwide. You can use Microsoft Graph to access notebooks, sections, and pages in OneNote to create solutions that help your users plan and organize ideas and information.
 
-### Collect and organize notes and ideas  
-Use OneNote as a canvas where users can add and arrange their content. Microsoft Graph makes it easy to write apps that enable students to take notes and do research, families to share plans and ideas, or shoppers to share pictures. Your app can grab the information people want, send it to OneNote, and then help them organize it.
-
-### Capture information in many formats
-Capture HTML, embed images (sourced locally or at a public URL), video, audio, email messages, and other common file types. OneNote can even render webpages and PDF files as snapshots. Microsoft Graph supports a set of standard HTML and CSS for OneNote page layout, so you can use tables, inline images, and basic formatting to get the look you want. 
-
-### Use the OneNote ecosystem to enhance your core scenarios
```

Comment:
IMO the imperative without to is a pretty typical perspective to use in value propositions. So I think the original "Use the OneNote ecosystem to enhance your core scenarios", "Reach millions of OneNote users on all major platforms" is appropriate.